src/validators/maxValue.js

```diff
@@ -0,0 +1,4 @@
+import { withParams } from './common'
+export default (max) => withParams(
+  { type: 'minValue', max },
+    value => value <= max)
```

Comment:
I like `minValue` and `maxValue` names. Could you fix the code style to match existing validators more? `minLength` is a good example.